src/core/Akka/Actor/Scheduler/HashedWheelTimerScheduler.cs

```diff
@@ -82,7 +81,7 @@ public HashedWheelTimerScheduler(Config scheduler, ILoggingAdapter log) : base(s
         /// <summary>
         /// 0 - init, 1 - started, 2 - shutdown
         /// </summary>
-        private volatile int _workerState = WORKER_STATE_INIT;
+        private int _workerState = WORKER_STATE_INIT;
```

Comment:
Guess it really did not need to be `volatile` if we're using `Interlocked` to access it primarily.

Reply:
yes, absolutely `Interlocked` has a memory barrier 
and `volatile` fields used often are a perf killer.

Best is to use `Violete.Write()`  for fields that are written only once or view times.